finder/tagged_test.go

```diff
@@ -19,25 +19,43 @@ func TestTaggedWhere(t *testing.T) {
 		// info about _tag "directory"
 		{"seriesByTag('key=value')", "Tag1='key=value'", "", false},
 		{"seriesByTag('name=rps')", "Tag1='__name__=rps'", "", false},
-		{"seriesByTag('name=~cpu.usage')", "Tag1 LIKE '\\\\_\\\\_name\\\\_\\\\_=cpu%' AND match(Tag1, '__name__=cpu.usage')", "Tag1 LIKE '\\\\_\\\\_name\\\\_\\\\_=cpu%' AND match(Tag1, '__name__=cpu.usage')", false},
+		{"seriesByTag('name=~cpu.usage')", "Tag1 LIKE '\\\\_\\\\_name\\\\_\\\\_=cpu%' AND match(Tag1, '^__name__=.*cpu.usage')", "Tag1 LIKE '\\\\_\\\\_name\\\\_\\\\_=cpu%' AND match(Tag1, '^__name__=.*cpu.usage')", false},
```

Comment:
Shouldn't it be `Tag1 LIKE '\\\\_\\\\_name\\\\_\\\\_=%cpu_usage%'`? LIKE and match() together look like nothing will be found

Reply:
Yes, it' a mistake. `Tag1 LIKE '\\\\_\\\\_name\\\\_\\\\_=%` in this case. 

`Tag1 LIKE '\\\\_\\\\_name\\\\_\\\\_=%cpu%` will also work, but not very useful for index check.